.dockerignore

```diff
@@ -3,3 +3,26 @@ node_modules
 **/node_modules
 **/dist
 dist
+tmp
+**/tmp
+
+docs
+infrastructure
+localenv
+
+.github
+.husky
+.dockerignore
+.eslintignore
+.eslintrc.yml
+.gitattributes
+.gitignore
+.prettierignore
+commitlint.config.js
+jest.config.base.js
+renovate.json
+
+**/jest.config.js
+**/jest.setup.js
+**/jest.teardown.js
+**/docker-compose.yml
```

Comment:
maybe we can also ignore test files in here?

Reply:
Yes!